gui/src/components/ClusterSummary.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <table class="table table-borderless text-left">
+        <thead>
+        <tr>
+            <th scope="col">Cluster ID</th>
```

Comment:
```suggestion
            <th scope="col">Cluster ID</th>
            <th scope="col">Cluster Name</th>
```
I just realized that cluster name should be here instead of in the title.